src/main/java/org/spongepowered/common/mixin/core/network/MixinNetHandlerPlayServer.java

```diff
@@ -95,6 +103,8 @@
 
     private Location lastMoveLocation = null;
 
+    private final Map<String, ResourcePack> sentResourcePacks = new HashMap<String, ResourcePack>();
```

Comment:
@Mumfrey Most of the time, yes, but not necessarily all of the time. [Consulting the Google Style Guide](https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s3.4.2-class-member-ordering) suggests that there is not a defined way to order members, but order them so that it is logical. To me, it's more logical to keep the fields close to the methods because in this particular mixin fields will not be shared outside the cluster of methods they are for.